tests/fuzz/s2n_client_cert_verify_recv_test.c

```diff
@@ -146,7 +146,7 @@ int s2n_fuzz_test(const uint8_t *buf, size_t len)
     struct s2n_connection *server_conn = s2n_connection_new(S2N_SERVER);
     POSIX_ENSURE_REF(server_conn);
     POSIX_GUARD(s2n_stuffer_write_bytes(&server_conn->handshake.io, buf, len));
-    server_conn->handshake_params.client_public_key.key.rsa_key.rsa = public_key.key.rsa_key.rsa;
+    server_conn->handshake_params.client_public_key.pkey = public_key.pkey;
```

Comment:
This wouldn't have been working for a while, even if it compiled. The fuzz test would have always trivially failed. I'm also not sure it's not still trivially failing for another reason. I just got it compiling again.

We badly need to revisit our fuzz tests.

Reply:
Time to look at the fuzz test coverage report?